app/Makefile.in

```diff
@@ -175,7 +176,8 @@ am__objects_2 = acvp_app-app_main.$(OBJEXT) acvp_app-app_aes.$(OBJEXT) \
 	acvp_app-app_sha.$(OBJEXT) acvp_app-app_utils.$(OBJEXT)
 @BUILD_APP_AS_LIB_FALSE@am_acvp_app_OBJECTS = $(am__objects_2)
 acvp_app_OBJECTS = $(am_acvp_app_OBJECTS)
-@BUILD_APP_AS_LIB_FALSE@@USE_FOM_OBJ_TRUE@acvp_app_DEPENDENCIES = $(FOM_OBJ_DIR)/fipscanister.o
+@BUILD_APP_AS_LIB_FALSE@acvp_app_DEPENDENCIES = $(am__DEPENDENCIES_1) \
+@BUILD_APP_AS_LIB_FALSE@	$(am__append_4)
```

Comment:
why all the spaces ?

Reply:
Generated by autoreconf, not sure if that is them doing their own thing or I shouldn't be using spaces around equal signs in the .am files; I will look into it for a later PR.